Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -590,6 +590,9 @@ for (const auto & particle_diag : particle_diags) {
     if ( particle_diag.m_plot_phi ) {
         storePhiOnParticles( tmp, WarpX::electrostatic_solver_id, !use_pinned_pc );
```

Comment:
@RemiLehe , do you know why for `storePhiOnParticles` we pass `!use_pinned_pc` instead of `full_diag` ?